src/modules/dashboard/components/SubscribedColoniesList/SubscribedColoniesList.css

```diff
@@ -122,19 +117,19 @@
     box-shadow: 0px 1px 6px var(--drop-shadow);
   }
 
-  .scrollableContainer > div[role="tooltip"] {
+  .scrollableContainer > div[role='tooltip'] {
```

Comment:
Is this the auto fix of your linter? I think we should keep it consistent with how it is in the dapp now and it seems it's double quotes.